client/js/store/modules/auth.js

```diff
@@ -107,5 +107,51 @@ export default {
         );
       });
     },
+    async requestStatus(context) {
+      if (!window) {
+        // Swallow error, this call is only supported client side
+        return;
+      }
```

Comment:
```suggestion
```
My thought is we leave it out until a time where it's relevant. By the time we more seriously consider SSR it's possible other things will have changed such that this will be stale. I'm not strongly opinionated about it, so I'll leave it to your judgement but my thought is we call any SSR considerations out of scope until we tackle it holistically.